.github/workflows/weekly_vuln_scan.yml

```diff
@@ -1,41 +1,60 @@
+# SPDX-FileCopyrightText: 2024 PNED G.I.E.
+# SPDX-License-Identifier: Apache-2.0
+
 name: Scan Latest Two Semantically Versioned Docker Image Tags with ORT
 
 on:
   schedule:
-    - cron: "0 14 * * 1" # Every Monday at 14:00 UTC
+    - cron: "0 14 * * 1" # Works on Every Monday at 14:00 UTC
   workflow_dispatch:
 
 jobs:
   fetch-and-scan:
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
       - name: Setup Docker
         run: |
           sudo service docker start
+          docker ps
```

Comment:
question: why do you need show the processes?

Reply:
It's only for understanding of docker daemon is working or not but I can remove it also.